tools/config/project.config.ts

```diff
@@ -7,21 +8,19 @@ export class ProjectConfig extends SeedConfig {
   constructor() {
     super();
     // this.APP_TITLE = 'Put name of your app here';
-    let additional_deps: Array<any> = [
+    let additional_deps: InjectableDependency[] = [
       // {src: 'jquery/dist/jquery.min.js', inject: 'libs'},
       // {src: 'lodash/lodash.min.js', inject: 'libs'},
     ];
 
-    this.DEV_NPM_DEPENDENCIES = this.DEV_DEPENDENCIES.concat(normalizeDependencies(additional_deps));
-    this.PROD_NPM_DEPENDENCIES = this.PROD_NPM_DEPENDENCIES.concat(normalizeDependencies(additional_deps));
+    const oldDeps = this.NPM_DEPENDENCIES;
```

Comment:
I get what we are doing here but is there a better word than "oldDeps" to convey intent? I can see the future issue questions already.